CHANGELOG.md

```diff
@@ -9,6 +9,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ...
 
+## 3.1.0 - 2020-02-26
+
+Add support for shims/scripts wrapping the `buildkite-agent` binary, instead of directly calling the binary.
```

Comment:
For example, if you might use the shim to wrap bk so you can fetch a registration token from vault.